src/lib/formats/dvk_mx_dsk.h

```diff
@@ -15,21 +15,26 @@
 class dvk_mx_format : public floppy_image_format_t
 {
 public:
+	static constexpr unsigned MX_SECTORS = 11;
+	static constexpr unsigned MX_SECTOR_SIZE = 256;
+
 	dvk_mx_format();
 
 	virtual int identify(util::random_read &io, uint32_t form_factor, const std::vector<uint32_t> &variants) const override;
 	virtual bool load(util::random_read &io, uint32_t form_factor, const std::vector<uint32_t> &variants, floppy_image &image) const override;
+	virtual bool save(util::random_read_write &io, const std::vector<uint32_t> &variants, const floppy_image &image) const override;
 
-	virtual const char *name() const noexcept override;
-	virtual const char *description() const noexcept override;
-	virtual const char *extensions() const noexcept override;
-	virtual bool supports_save() const noexcept override;
+	const char *name() const noexcept override { return "mx"; }
+	const char *description() const noexcept override { return "DVK MX disk image"; }
+	const char *extensions() const noexcept override { return "mx"; }
+	bool supports_save() const noexcept override { return true; }
```

Comment:
We usually explicitly specify `virtual` on overridden virtual member functions.  Note that the other virtual member functions in this header do so.